### PR TITLE
Update maven pipelines

### PR DESCRIPTION
**Pull request by arilivigni**
* Release, ReleaseandStage, ReleaseStageApproveAndPromote
  * Name and description
* Change from Build release to Build Image
* Add shell:'/bin/bash'

### CODE REVIEW

**Comment by piyush-garg** [comment]
@arilivigni Changes look good to me, we need that bash shell for upstream images but to get that working you need to change the plugin version also here https://github.com/fabric8io/fabric8-pipeline-library/blob/master/pom.xml#L265 AFAIK

Are you able to get it working?

**Comment by arilivigni** [comment]
@piyush-garg I think ultimately we want to go the way plugins get installed the OpenShift way.  When I looked before we are doing this differently on our images.  The Jenkins master should be based on the OpenShift with mods to init.groovy, plugin installation, etc.  We can discuss in more detail.

**Comment by sthaha** [comment]
@arilivigni I merged @piyush-garg 's patch that updates the description, so could you please resolve the conflict and resubmit?

**Comment by arilivigni** [comment]
@rupalibehera done shall I merge?

File: maven/ReleaseAndStage/Jenkinsfile [hunk 1]
```diff
@@ -37,8 +37,8 @@ mavenNode {
     }
 
     echo 'NOTE: running pipelines for the first time will take longer as build and base docker images are pulled onto the node'
-    container(name: 'maven') {
-      stage('Build Image') {
+    container(name: 'maven', shell:'/bin/bash') {
+      stage('Build Release') {
```
Comment (piyush-garg) [inline maven/ReleaseAndStage/Jenkinsfile]: you need to change it Release -> Image

Comment (arilivigni) [reply maven/ReleaseAndStage/Jenkinsfile]: Why does this need to change?  It worked in OSIO with the fabric8 image and ran correctly.

Comment (piyush-garg) [reply maven/ReleaseAndStage/Jenkinsfile]: I mean the stage name. That is not required to change

Comment (arilivigni) [reply maven/ReleaseAndStage/Jenkinsfile]: Will change to image

**Comment by arilivigni** [comment]
@piyush-garg fixed rebase issue good catch thanks!

**Review by rupalibehera — APPROVED** [review]

**Pull request merged**